video/cloud-client/analyze/beta_snippets.py

```diff
@@ -55,21 +55,26 @@ def speech_transcription(input_uri):
     # There is only one annotation_result since only
     # one video is processed.
     annotation_results = result.annotation_results[0]
-    speech_transcription = annotation_results.speech_transcriptions[0]
-    alternative = speech_transcription.alternatives[0]
-
-    print('Transcript: {}'.format(alternative.transcript))
-    print('Confidence: {}\n'.format(alternative.confidence))
-
-    print('Word level information:')
-    for word_info in alternative.words:
-        word = word_info.word
-        start_time = word_info.start_time
-        end_time = word_info.end_time
-        print('\t{}s - {}s: {}'.format(
-            start_time.seconds + start_time.nanos * 1e-9,
-            end_time.seconds + end_time.nanos * 1e-9,
-            word))
+    for speech_transcription in annotation_results.speech_transcriptions:
+
+        # The number of alternatives for each transcription is limited by
```

Comment:
Would it also be good to state what an alternative is?
My understanding from the name is that there are several different guesses at how to transcribe this video here are our different results of what we think this transcript is during this time frame?

Reply:
Added a few words.